rust/operator-binary/src/airflow_controller.rs

```diff
@@ -1073,7 +1073,7 @@ fn build_executor_template_config_map(
         .context(GracefulShutdownSnafu)?;
 
     // N.B. this "base" name is an airflow requirement and should not be changed!
-    // See https://airflow.apache.org/docs/apache-airflow/2.6.1/core-concepts/executor/kubernetes.html#base-image
+    // See https://airflow.apache.org/docs/apache-airflow-providers-cncf-kubernetes/stable/kubernetes_executor.html#base-image
```

Comment:
I think we should link to a specific version to avoid a broken link in the future
```suggestion
    // See https://airflow.apache.org/docs/apache-airflow-providers-cncf-kubernetes/8.4.0/kubernetes_executor.html#base-image
```

Reply:
Good catch